go/vt/tabletserver/sysloglogger/sysloglogger.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2016, Google Inc. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package sysloglogger
```

Comment:
Please add a comment before this line of the form:

// Package sysloglogger ....    where you describe what it does